### ISSUE TITLE
Inability to set integration up

### ISSUE THREAD

**Issue by lankthedank** [gndean/home-assistant-hypervolt-charger#37]
I am unable to set the integration up from scratch, even after reinstalling it multiple times. On a previous Home Assistant installation (2023.10.x), the integration worked flawlessly, aside from #29 which seemed to have resolved itself.

This issue is present on 2023.12.0 and 2023.11.3. I am running Home Assistant Supervised, integration version 1.3.0 installed through HACS. The HASS installation is fairly fresh with only a few additional integrations installed.

Surrounding logs prior to the below error (final 10 digits of ID hidden):
```
2023-12-06 22:36:25.829 INFO (MainThread) [custom_components.hypervolt_charger.hypervolt_api_client] Login loading URL: https://auth.hypervolt.co.uk/authorize
2023-12-06 22:36:26.963 INFO (MainThread) [custom_components.hypervolt_charger.hypervolt_api_client] HypervoltApiClient logged in!
2023-12-06 22:36:27.085 INFO (MainThread) [custom_components.hypervolt_charger.config_flow] Found chargers: [{'charger_id': 21645485##########, 'created': '2023-02-04T15:16:51.120127Z'}, {'charger_id': 7233931##########, 'created': '2023-05-12T19:10:20.457540Z'}, {'charger_id': 20248##########, 'created': '2023-05-13T11:20:14.603372Z'}]
2023-12-06 22:36:27.088 ERROR (MainThread) [aiohttp.server] Error handling request
```

Error:
```
Logger: aiohttp.server
Source: /usr/local/lib/python3.11/site-packages/aiohttp/web_protocol.py:421
First occurred: 10:36:27 PM (1 occurrences)
Last logged: 10:36:27 PM

Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/aiohttp/web_protocol.py", line 452, in _handle_request
    resp = await request_handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiohttp/web_app.py", line 543, in _handle
    resp = await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/aiohttp/web_middlewares.py", line 114, in impl
    return await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 85, in security_filter_middleware
    return await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 227, in forwarded_middleware
    return await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
    return await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 80, in ban_middleware
    return await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 233, in auth_middleware
    return await handler(request)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/headers.py", line 31, in headers_middleware
    response = await handler(request)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 149, in handle
    result = await handler(request, **request.match_info)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/decorators.py", line 63, in with_admin
    return await func(self, request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 177, in post
    return await super().post(request, flow_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 72, in wrapper
    result = await method(view, request, data, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 110, in post
    result = await self._flow_mgr.async_configure(flow_id, data)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 293, in async_configure
    result = await self._async_handle_step(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 417, in _async_handle_step
    self._raise_if_step_does_not_exist(flow, result["step_id"])
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 440, in _raise_if_step_does_not_exist
    raise UnknownStep(
homeassistant.data_entry_flow.UnknownStep: Handler ConfigEntriesFlowManager doesn't support step select_charger

```
There are no subsequent logs.

**Comment by gndean** [comment]
I'm not sure what's happening here. It's maybe something related to having multiple chargers and a change in HA that only affects the config flow when having more than one charger. But, on scanning the HA release notes, I can't see what that would be. Did you manage to resolve this?

**Comment by lankthedank** [comment]
I resolved it by removing the flow that allows adding the chargers in bulk, instead rerunning the flow with each charger's list index hard-coded.

It's terrible and not a solution, but I wasn't able to work out what was wrong in the first place. I haven't got any experience with writing integrations, so I'm pretty unfamiliar with how the config_flow is handled by HA.

It makes sense to have been a change in HA because it only cropped up after a reinstall requiring the integration to be set back up, though I couldn't find what the breaking change was either, or which version caused it.

**Comment by gndean** [comment]
This should be fixed in V1.3.1 now. Thanks for your support
